### ISSUE TITLE
Phylum and Division both exist by default for botanical collections

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#3424]
![image](https://user-images.githubusercontent.com/37256050/234939009-a848e676-dc01-4368-9682-5c77cc5deba6.png)

Specify 6 automatically hides the `Division` rank for Botanical collections and uses `Phylum` instead.
Specify 7 does not do this, so you have both ranks duplicated.

This is problematic in multiple ways, including in https://github.com/specify/specify7/issues/3340.

RankID is the same for both, which is why you experience the problem.

**Comment by maxxxxxdlp** [comment]
Doubt this will be fixed until https://github.com/specify/specify7/issues/2932

**Comment by grantfitzsimmons** [comment]
@maxxxxxdlp Yes it is, but Specify 7 should not display both. Encountered again with Jerusalem.

**Comment by grantfitzsimmons** [comment]
Ranks with the same RankID should not be shown twice

**Comment by maxxxxxdlp** [comment]
Ah, now that you say that, it's a relatively simple solution to implement

this is the place where tree ranks are fetched
https://github.com/specify/specify7/blob/207df9a091c59cd04e4e4b1c3eb111c75c74f68a/specifyweb/frontend/js_src/lib/components/InitialContext/treeRanks.ts#L58-L104

if I remember correctly, this is the only place where front-end fetches tree ranks, and all others use this place

just have to update this script to remove duplicate ranks.

quite simple to implment, assuming you are ok with us just picking the first matching rank, rather than doing smart choosing based on the current discipline

**Comment by emenslin** [comment]
Can recreate in edge (7.9.6)
